src/main/java/com/palpal/dealightbe/domain/order/application/dto/response/OrderRes.java

```diff
@@ -0,0 +1,28 @@
+package com.palpal.dealightbe.domain.order.application.dto.response;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+import com.palpal.dealightbe.domain.order.domain.Order;
+
+public record OrderRes(
+
+	Long orderId,
```

Comment:
```suggestion
public record OrderRes(
	Long orderId,
```
개행 제거 해주시면 될 것 같습니다!